src/vm/reflectioninvocation.cpp

```diff
@@ -1383,8 +1383,18 @@ FCIMPL4(Object*, RuntimeMethodHandle::InvokeMethod,
         }
 #endif
 
+#ifdef _TARGET_ARM64_
+        ArgLocDesc argLocDescForStructInRegs;
```

Comment:
Yes and that seems like unification with unix amd64 which we decided to do later and not as part of this work.